packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -178,16 +178,17 @@ export class AndroidEmulatorDevice extends DeviceBase {
         "location_mode",
         "3",
       ]);
-      // note that geo fix command takes arguments: $longitude , $latitude so the order is reversed compared to most conventions
-      await exec(ADB_PATH, [
-        "-s",
-        this.serial!,
-        "emu",
-        "geo",
-        "fix",
-        settings.location.longitude.toString(),
-        settings.location.latitude.toString(),
-      ]);
+
+      // This is a work around for the problem with emu geo command not working  when passed, 0 0 coordinates
+      // when provided coordinates are close enough to 0 that the adb assumes they are 0 we pass the smallest
+      // working number instead. Moreover note that geo fix command takes arguments:
+      // $longitude , $latitude so the order is reversed compared to most conventions
+      const areCoordinatesToCloseToZero =
```

Comment:
does the adb command work with exactly one coordinate equal to `0`? bc here you have `&&` so if user inputs `0 1000`, the check is false and we pass `0 1000` to adb

Reply:
@km1chno yes it works with just 1 zero